test/dlc_tests/conftest.py

```diff
@@ -335,14 +332,18 @@ def generate_unique_values_for_fixtures(metafunc_obj, images_to_parametrize, val
 
                     image_tag = image.split(":")[-1].replace(".", "-")
 
-                    framework = image.split(":")[0].split("/")[1].split("-")[0]
```

Comment:
note: this was not actually grabbing framework and job_type - simplified implementation so we won't run into the mxnet1.6/pytorch1.6 bugs